src/atom-environment.coffee

```diff
@@ -885,8 +885,6 @@ class AtomEnvironment extends Model
         else
           @project.addPath(pathToOpen)
 
-      @applicationDelegate.addRecentDocument(pathToOpen)
-
```

Comment:
@DouweM I see your point about wanting to use the recent documents list as something of a 'Recent Projects' list, but I don't think it makes sense to use the recent documents list to represent that concept.

With the previous code, whether or not a file got added to the recent documents list depended on an implementation detail of *how* the file was opened. Opening a file with `command-o` would add it, but opening the same file with `command-p` would not. Opening your stylesheet with the `Application: Open Your Stylesheet` command would add it, but adding a root folder with `command-shift-o` would not add it.

Also, Atom's concept of a 'Project' can in general consist of multiple root folders. So the objects that were added to the recent documents list didn't correspond to projects consistently either.

You might want to use the [recent-projects](https://github.com/EXSyst/recent-projects) package for the workflow that you're describing.